src/main/java/org/example/exception/OwnerHasNoPropertyException.java

```diff
@@ -0,0 +1,11 @@
+package org.example.exception;
+
+public class OwnerHasNoPropertyException extends Exception{
```

Comment:
Exceptions should have smth different except for name. Try to make something special about each exception.